dist/dfu.js

```diff
@@ -134,6 +134,9 @@
             })
             .then(function() {
                 log("modeData written");
+                setTimeout(function() {  // TODO: Remove this when gattserverdisconnected event is implemented and possibly put a timeout in that event handler before resolving.
```

Comment:
I won't be able to now, I need to bring in my Macbook from home to test node. I'll test it tomorrow as long as i remember to bring it with me :p